### PR TITLE
Bluetooth: host: Run scan/slow test on nRF53

### PR DESCRIPTION
**Pull request by jori-nordic**
This enables the bluetooth/host/scan/slow test on nRF53 in the Zephyr
Continuous Integration pipeline.

Why waste more CI cycles?

The 5340 build will:
1. stress-test the HCI IPC driver (controller + host sides)
2. use a different buffer allocation model, where the allocator doesn't
know what other events are in the queue

`2.` is especially important, as this is the model a lot of host users
will be running with. Any controller driver that is not directly on-chip
_cannot_ know what is in the HCI queues, and cannot take the same
convenient shortcuts as the on-chip function-called drivers.

Stacked PR on top of:
- https://github.com/zephyrproject-rtos/zephyr/pull/78397

Blocked on
- https://github.com/zephyrproject-rtos/zephyr/issues/50786

### CODE REVIEW

**Review by aescolar — APPROVED** [review]
No complaints from me

File: tests/bsim/bluetooth/host/scan/slow/sysbuild/hci_ipc.conf
Comment (Thalley) [inline tests/bsim/bluetooth/host/scan/slow/sysbuild/hci_ipc.conf:1]: Is this necessary? 

Do we need extended advertising for this test?
And if so, should we consider an non-ISO overlay for extended advertising in the sample itself (ping @cvinayak)?

Comment (jori-nordic) [reply tests/bsim/bluetooth/host/scan/slow/sysbuild/hci_ipc.conf:1]: yes extended is crucial, as without it, the host will discard reports thus hiding the issue. 
Extended is used to repro https://github.com/zephyrproject-rtos/zephyr/issues/50786 basically.

Comment (Thalley) [reply tests/bsim/bluetooth/host/scan/slow/sysbuild/hci_ipc.conf:1]: Right :) 

I don't see this file being added to the build explicitly; is it implicitly included when building the netcore when using sysbuild since it's in a `sysbuild` directory? Or is the name `hci_ipc.conf` the reason why it's in the build?

Comment (cvinayak) [reply tests/bsim/bluetooth/host/scan/slow/sysbuild/hci_ipc.conf:1]: > Is this necessary?
> 
> Do we need extended advertising for this test? And if so, should we consider an non-ISO overlay for extended advertising in the sample itself (ping @cvinayak)?

I am not well verse with sysbuild; maybe the current way in PR is common to Zephyr Controller and SoftDevice Controller under BabbleSIM ?

Comment (Thalley) [reply tests/bsim/bluetooth/host/scan/slow/sysbuild/hci_ipc.conf:1]: Thanks - Learned something new today :) I guess there are some sysbuild documentation somewhere that I should read sometime soon to learn the magic tricks

**Review by Thalley — COMMENTED** [review]
Will approve once the PR this PR depends on is merged (suggest to add DNM on this until then, to avoid accidentally merging this before the other)

**Comment by jori-nordic** [comment]
> suggest to add DNM

no need, the CI does the needful :)
image

File: tests/bsim/bluetooth/host/scan/slow/sysbuild/hci_ipc.conf
Comment (jori-nordic) [reply tests/bsim/bluetooth/host/scan/slow/sysbuild/hci_ipc.conf:1]: yup. It's actually more straightforward than reading the old `child_image` stuff we had in ncs: https://docs.zephyrproject.org/latest/build/sysbuild/index.html#zephyr-application-kconfig-fragment-and-devicetree-overlay

**Pull request closed without merging**
